etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -26,6 +26,12 @@ def __init__(
         output_dims: int = 320,
         hidden_dims: int = 64,
         depth: int = 10,
+        encoding_window: Optional[Union[Literal["multiscale"], int]] = None,
```

Comment:
I return encode params in constructor of model because ts2vec и tstcс have different encode params (tstcc doesnt have at all) and they cant be passed to transform

Reply:
We should discuss possible solutions for this problem. For example, we could introduce in transform parameter like: `encoding_params` or smth like this. Probably, there are also other solutions.